src/view/Dialog.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import './Dialog.less';
+import {DialogData} from "../utils/typer";
+import {Innholdstittel} from "nav-frontend-typografi";
+import {Showdialog} from "./Showdialog";
+
+interface Props {
+    dialog: DialogData | null;
+}
+
+export function Dialog(props: Props) {
+    if (props.dialog !== null) {
+        return (
+            <Showdialog dialog={props.dialog}/>
+        )
+    } else {
+        return (
+            <div className="dialog">
+                <div className="dialog-top">
+                    <hr/>
```

Comment:
`hr` har semantisk betydning i html, som også vil bety at den leses opp for de med skjermleser. Om formålet kunn er å ha et visuelt skille så er det bedre å bruke `border`-attributter

Reply:
gjort